src/System.Runtime.Serialization.Xml/tests/System.Runtime.Serialization.Xml.Tests.csproj

```diff
@@ -3,6 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <ProjectGuid>{30CAB353-089E-4294-B23B-F2DD1D945654}</ProjectGuid>
+    <DefineConstants Condition="'$(TargetGroup)' == 'uapaot'">$(DefineConstants);uapaot</DefineConstants>
```

Comment:
Yeah, I also noticed that other projects failed due to including this file. Maybe we can remove the test and the test type entirely for now and add them back later.